cds-snc/cppd-medical-report-sails
# Initial Migrations for DB

These scripts should create a database. 

## To Test 

Start a database
Change `AUTO_MIGRATE_MODE` in the `.env` file to "safe"
Run `npm run db:migrate` to run the migrations
Run `npm run dev` to start the project
Run `npm run cypress:open` to open cypress then run the tests.
Make sure all tests pass

Also just play around make sure nothing breaks.
